com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -36,8 +37,9 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Removed `FixedQueue`, `StreamExtensions`, `TypeExtensions` (#1398)
 
 ### Fixed
+
 - Fixed in-scene NetworkObjects that are moved into the DDOL scene not getting restored to their original active state (enabled/disabled) after a full scene transition (#1354)
-- Fixed invalid IL code being generated when using `this` instead of `this ref` for the FastBufferReader/FastBufferWriter parameter of an extension method. (#1393)
+- Fixed invalid IL code being generated when using this instead of this ref for the FastBufferReader/FastBufferWriter parameter of an extension method. (#1393)
```

Comment:
👀

Reply:
o.o wat... the diff didn't show that before I checked it in